src/Elliptic/Systems/Elasticity/Equations.cpp

```diff
@@ -1,38 +1,37 @@
 // Distributed under the MIT License.
 // See LICENSE.txt for details.
 
-#include "Elliptic/Systems/Elasticity/Equations.hpp"
-#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
-
 #include <cstddef>
 
 #include "DataStructures/DataVector.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/Systems/Elasticity/Equations.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
-#include "Utilities/MakeWithValue.hpp"
 
 namespace Elasticity {
 
 template <size_t Dim>
 void primal_fluxes(
-    const gsl::not_null<tnsr::IJ<DataVector, Dim, Frame::Inertial>*>
-        flux_for_displacement,
-    const tnsr::ii<DataVector, Dim, Frame::Inertial>& strain,
+    const gsl::not_null<tnsr::IJ<DataVector, Dim>*> flux_for_displacement,
+    const tnsr::ii<DataVector, Dim>& strain,
     const ConstitutiveRelations::ConstitutiveRelation<Dim>&
         constitutive_relation,
     const tnsr::I<DataVector, Dim>& coordinates) noexcept {
-  auto stress = constitutive_relation.stress(strain, coordinates);
+  const auto stress = constitutive_relation.stress(strain, coordinates);
+  // The stress is symmetric, due to eucledian geometry in contrast to the more
+  // general flux_for_displacement
```

Comment:
Please make this comment more descriptive so that it answers the following questions:
- Why can't you `std::move(stress.get(d, e))` below? (each component's data is needed twice because `stress` is symmetric but the flux tensor type isn't)
- Why can't we do something like `flux_for_displacement = -stress`? (we don't have a `-` operator for tensors yet, but even if we did the tensor types are different for two reasons: (1) symmetry for technical reasons, (2) the flux has upper indices and the stress lower, here you implicitly assume a Euclidean geometry)

(from https://github.com/sxs-collaboration/spectre/pull/1927#discussion_r371998423)

Please also add a note on the minus sign (the `stress` function computes `-Y^ijkl S_kl`).